docs/sdks/server-sdks/ruby.md

```diff
@@ -19,33 +21,75 @@ gem install eppo-server-sdk
 or add to you `Gemfile`:
 
 ```
-gem 'eppo-server-sdk', '~> 0.3.0'
+gem 'eppo-server-sdk', '~> 3.0.0'
 ```
 
-## 2. Initialize the SDK
+### B. Initialize the SDK
 
-Initialize the SDK with a SDK key, which can be generated in the Eppo interface. Initialization the SDK when your application starts up to generate a singleton client instance, once per application lifecycle:
+To initialize the SDK, you will need an SDK key. You can generate one [in the flag interface](https://eppo.cloud/feature-flags/keys).
```

Comment:
minor, but would it be better to call it the "flags" interface (plural), or "configuration" as that's what they click to expose the SDK keys tab. OR we could just link to that!

![image](https://github.com/user-attachments/assets/80242e1d-51c5-4106-a6a2-f044f89e0c48)